tests/test_cli.py

```diff
@@ -352,12 +382,21 @@ def should_print_version_import_status_verbatim(
         ASSET_UPLOAD_KEY: {STATUS_KEY: Outcome.SKIPPED.value, ERRORS_KEY: []},
     }
     response_object = get_response_object(HTTPStatus.OK, response_body)
+    response_encode = dumps(response_object).encode()
+    response_payload = StreamingBody(io.BytesIO(response_encode), len(response_encode))
```

Comment:
At this point we might want to pack the response generation into a function, to hide the gnarly details.